cadl-tests/cadl/auth.cadl

```diff
@@ -0,0 +1,22 @@
+import "@cadl-lang/rest";
+
+using Cadl.Http;
+
+@serviceTitle("Auth")
+@serviceVersion("1.0.0")
+@useAuth(OAuth2Auth<[AuthFlow]> | ApiKeyAuth<ApiKeyLocation.header, "x-ms-api-key">)
+namespace Cadl.Auth;
+
+model AuthFlow {
+    type: OAuth2FlowType.clientCredentials;
+    tokenUrl: "https://api.example.com/oauth2/token";
```

Comment:
are `tokenUrl` and `refreshUrl` used in generated code?

Reply:
No. They are done in azure-identity. These value in CADL won't take effect.